.buildkite/publish/run.sh

```diff
@@ -2,27 +2,37 @@
 
 set -ex
 
+# Install pnpm
 npm i --silent -g pnpm@6 --unsafe-perm # TODO: is this unsafe-perm needed?
 
+# Install packages
 pnpm i
 
-pnpm run lint
-
+# Output versions
+pnpm -v
 node -v
 npm -v
 
+# See package.json setup script
 pnpm run setup
 
-# TODO: why is this necessary?
-cd packages/integration-tests
-pnpm i sqlite3@5.0.2 --unsafe-perm --reporter=silent
-cd ../..
+if [[ $BUILDKITE_BRANCH == integration/* ]] ;
```

Comment:
Perhaps we can make this happen at the very beginning?

Reply:
What does `the very beginning` means 😅 

I don't see where I can put it earlier in this case